content/pt/docs/concepts/configuration/pod-overhead.md

```diff
@@ -0,0 +1,197 @@
+---
+reviewers:
+- dchen1107
+- egernst
+- tallclair
+title: Pod Overhead
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
+
+Quando executa um Pod num nó, o próprio Pod usa uma quantidade de recursos do sistema. Estes
+recursos são adicionais aos recursos necessários para executar o(s) _container(s)_ dentro do Pod.
+Sobrecarga de Pod, do inglês _Pod Overhead_, é uma funcionalidade que serve para contabilizar os recursos consumidos pela
+infraestrutura do Pod para além das solicitações e limites do _container_.
+
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+No Kubernetes, a sobrecarga de _Pods_ é definido no tempo de
+[admissão](/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks)
+de acordo com a sobrecarga associada à
+[RuntimeClass](/docs/concepts/containers/runtime-class/) do _Pod_.
+
+Quando é ativada a Sobrecarga de Pod, a sobrecarga é considerada adicionalmente à soma dos 
+pedidos de recursos do _container_ aquando do agendamento de um Pod. Semelhantemente, o _kubelet_
+vai incluir a sobrecarga do Pod aquando do dimensionamento do cgroup do Pod, e aquando de
+levar a cabo a classificação de despejo do Pod.
+
+## Possibilitando a Sobrecarga do Pod {#set-up}
+
+Terá de garantir que o [portão de funcionalidade](/docs/reference/command-line-tools-reference/feature-gates/)
+`PodOverhead` está ativo (está ativo por defeito a partir da versão 1.18)
+por todo o cluster, e uma `RuntimeClass` é utilizada que defina o campo `overhead`.
+
+## Exemplo de uso
+
+Para usar a funcionalidade PodOverhead, é necessário uma RuntimeClass que define o campo `overhead`.
+Por exemplo, poderia usar a definição da RuntimeClass abaixo com um _container runtime_ virtualizado
+que usa cerca de 120MiB por Pod para a máquina virutal e o sistema operativo convidado:
```

Comment:
```suggestion
que usa cerca de 120MiB por Pod para a máquina virtual e o sistema operacional convidado:
```

Reply:
Fixed "virtual", will keep operativo as of pt_pt.